Locales.lua

```diff
@@ -2035,6 +2035,10 @@ L["Frozen Coffer"] = true
 L["Obsidian Battle Horn"] = true
 L["Emmah"] = true
 L["Box of Rattling Chains"] = true
+L["Brul"] = true
+L["Chest of Massive Gains"] = true
+L['Looted from "Chest of Massive Gains" which spawns after killing rare - Brullo the Strong. Loot chest while "Rocks on the Rocks" buff is active. Buff is obtained from eating food "Rocks on the Rocks"'] =
```

Comment:
The chest, item, and NPC names should likely be separate phrases. This allows re-using them without requiring a re-translation, and (more importantly) we can already submit the translated phrases by copy/pasting from wowhead.